<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Escape</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="CaseWidget" name="A3widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CaseWidget" name="B2widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CaseWidget" name="A2widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="CaseWidget" name="A1widget" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CaseWidget" name="B1widget" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="CaseWidget" name="C2widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="CaseWidget" name="B3widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="CaseWidget" name="C3widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="CaseWidget" name="C1widget" native="true">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecommencer"/>
   <addaction name="actionOuvrir_image"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecommencer">
   <property name="text">
    <string>Recommencer</string>
   </property>
  </action>
  <action name="actionOuvrir_image">
   <property name="text">
    <string>Ouvrir image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CaseWidget</class>
   <extends>QWidget</extends>
   <header location="global">casewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
